test/local-development.test.js

```diff
@@ -56,12 +56,13 @@ describe(`Local setup ${isExtended ? '*extended*' : ''}`, () => {
     if (isExtended) {
       await openAddon(page);
     } else {
-      await page.goto('https://localhost:3000/dialog-demo-bootstrap.html');
+      await page.goto('https://localhost:3000/dialog-demo-bootstrap-impl.html');
```

Comment:
Switching to loading the actual implementation of the react app rather than the wrapper for these "basic" tests, since the wrapper is no longer being served during development.

This also requires changing the triggering webpack text on line 32, as well as increasing the height of the `html` element to match the wrapper size. Otherwise snapshots would be much smaller in height and result in greater percent difference between OSs.